fpage/templates/fpage/home.html

```diff
@@ -0,0 +1,203 @@
+                </a>
+            </li>
+            <li class="mr-2 nav-item">
+                <a class="nav-link active " href="{% url 'fpage:song-list' %}">
+                    <span class="fa fa-headphones"></span> Songs
+                </a>
+            </li>
+            <li class="mr-2 nav-item active">
+                <a class="nav-link" href="{% url 'fpage:album-list' %}">
+                    <span class="fa fa-plus"></span> Add Albums
+                </a>
+            </li>
+            {% if request.user.is_authenticated %}
+            <li class="mr-2 nav-item dropdown active">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
+                    aria-haspopup="true" aria-expanded="false">
+                    {{ request.user }}
+                </a>
+                <div class="dropdown-menu " aria-labelledby="navbarDropdown">
+                    <a class="dropdown-item active" href="{% url 'fpage:fav-album-list' %}">Favourite Albums</a>
+                    <a class="dropdown-item" href="{% url 'fpage:fav-song-list' %}">Favourite Songs</a>
+                    <a class="dropdown-item" href="{% url 'fpage:logout' %}">Logout</a>
+                </div>
+            </li>
+            {% else %}
+            <li class="mr-2 nav-item">
+                <a class="nav-link active" href="{% url 'fpage:login' %}">Login</a>
+            </li>
+            <li class="mr-2 nav-item">
+                <a class="nav-link  active" href="{% url 'fpage:register' %}">Register</a>
+            </li>
+            {% endif %}
+        </ul>
+        {% if request.user.is_authenticated %}
+        <form class="form-inline my-2 my-lg-0">
+            <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+            <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+        </form>
+        {% endif %}
+    </div>
+</nav>
+
+
+<body>
+
+  <!-- Header -->
+  <header class="py-5 mb-5 home ">
+    <div class="container h-100">
+      <div class="row h-100 align-items-center">
+        <div class="col-lg-12">
+          <h1 class="display-4 text-white mt-5 mb-2">Albums</h1>
+          <p class="lead mb-5 text-white-50 animated bounce">Want to remember a song or Album? Dont worry Albums got it covered......</p>
+        </div>
+      </div>
+    </div>
+  </header>
+
+  <!-- Page Content -->
+  <div class="container">
+
+    <div class="row">
+      <div class="col-md-8 mb-5">
+        <h2>Features</h2>
+        <hr>
+        <p>Albums provides user with many features such as adding songs and albums, favouriting songs and albums,listen later feature , keeping History of all your listened songs and many more!</p>
+        <a class="btn btn-primary btn-lg" href="{% url 'fpage:register' %}">Register &raquo;</a>
+      </div>
+
+    </div>
+    <!-- /.row -->
+
+    <div class="card-deck">
```

Comment:
Card deck is not a good idea when it comes to responsiveness of the website. Use bootstrap grid instead. Place the cards inside columns inside row. 

Sample code:
```
<div class=container>
    <div class="row">
        <!-- The col classes will depend on your thinking of how the layout should look in various screen sizes -->
        <div class="col">
            <!-- Card here -->
        </div>
    </div>
</div>
```

Reply:
ok i didnt know about it. i will change it to bootstrap grid.

Reply:
when i use <div class="col"> the cards become horizontal

Reply:
You have to read about col classes in bootstrap docs to understand what type of classes you need for the layout. Also inside the col, you have make put another div with class=card and then put the a card title, body, content, footer etc inside it. If you miss the div with card class,
You will face errors.

Reply:
@ExpressHermes I changed it and its working properly but the images are not getting displayed i am using this ``` src="{ % static 'images/Song.jpeg' % }" ```

Reply:
![image](https://user-images.githubusercontent.com/45357209/97797771-c4805d00-1c45-11eb-820c-928b447a4809.png)
When i use that tag i get this as the output

Reply:
Add the `{% load static %}` at the top of the template in which images have been coded. If it still doesn't work, attach a screenshot of your code next time.

Reply:
no its still not working and as we are extending base.html we have loaded static there so it should extend that too

Reply:
```
{% extends "fpage/base.html" %}
{% load static %}
{% block content %}

<body>
  <!-- Header -->
  <header class="py-5 mb-5 home">
    <div class="container d-flex align-items-center home-container">
      <div class="row">
        <div class="col-lg-12">
          <h1 class="display-4 text-white mt-5 mb-2">Albums</h1>
          <p class="lead mb-5 text-white-50 animated bounce">Want to remember a song or Album? Dont worry Albums got it covered......</p>
        </div>
      </div>
    </div>
  </header>

  <!-- Page Content -->
  <div class=container>
    <div class="row">
      <div class="col-md-8 mb-5">
        <h2>Features</h2>
        <hr>
        <p>Albums provides user with many features such as adding songs and albums, favouriting songs and albums,listen later feature , keeping History of all your listened songs and many more!</p>
        <a class="btn btn-primary btn-lg" href="{% url 'fpage:register' %}">Register &raquo;</a>
      </hr>
      </div>
    </div>
    <div class="row mb-5">

        <div class="col">
            <div class="card card-color text-white" style="width: 20rem;">
		        <img class="card-img-top" src="{ % static 'images/Song.jpeg' % }" alt="Card image cap">
		        <div class="card-body">
		          <h5 class="card-title">Albums</h5>
		          <p class="card-text">Is it your birthday song album or aniversary or any album close to you keep it safe with us.</p>
		        </div>
		        <ul class="list-group list-group-flush">
		          <li class="list-group-item list-group-item bg-dark">Add,Delete or Modify ALbums</li>
		          <li class="list-group-item list-group-item bg-dark">Add Songs to your Albums</li>
		          <li class="list-group-item list-group-item bg-dark">Favourite you ALbum</li>
		        </ul>
		        <div class="card-body">
		            <a href="{% url 'fpage:login' %}" class="btn btn-info">Login</a>
		        </div>
      		</div>
        </div>

        <div class="col">
            <div class="card card-color text-white" style="width: 20rem;">
		        <img class="card-img-top" src="https://placehold.it/300x200" alt="Card image cap">
		        <div class="card-body">
		          <h5 class="card-title">Albums</h5>
		          <p class="card-text">Is it your birthday song album or aniversary or any album close to you keep it safe with us.</p>
		        </div>
		        <ul class="list-group list-group-flush">
		          <li class="list-group-item list-group-item bg-dark">Add,Delete or Modify ALbums</li>
		          <li class="list-group-item list-group-item bg-dark">Add Songs to your Albums</li>
		          <li class="list-group-item list-group-item bg-dark">Favourite you ALbum</li>
		        </ul>
		        <div class="card-body">
		            <a href="{% url 'fpage:login' %}" class="btn btn-info">Login</a>
		        </div>
      		</div>
        </div>

        <div class="col">
            <div class="card card-color text-white" style="width: 20rem;">
		        <img class="card-img-top" src="https://placehold.it/300x200" alt="Card image cap">
		        <div class="card-body">
		          <h5 class="card-title">Albums</h5>
		          <p class="card-text">Is it your birthday song album or aniversary or any album close to you keep it safe with us.</p>
		        </div>
		        <ul class="list-group list-group-flush">
		          <li class="list-group-item list-group-item bg-dark">Add,Delete or Modify ALbums</li>
		          <li class="list-group-item list-group-item bg-dark">Add Songs to your Albums</li>
		          <li class="list-group-item list-group-item bg-dark">Favourite you ALbum</li>
		        </ul>
		        <div class="card-body">
		            <a href="{% url 'fpage:login' %}" class="btn btn-info">Login</a>
		        </div>
      		</div>
        </div>
    </div>
</div>
  <!-- Bootstrap core JavaScript -->
  <script src="vendor/jquery/jquery.min.js"></script>
  <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

</body>


{% endblock %}
```

Reply:
Don't put any space between `{` and '%'.

Reply:
I had done the same back but when i wanst getting maybe due to load static i googled and SO said to put a space between ( and % so... . Anyways i have done it and i am making a commit . I have rerouted routes to get the homepage when we logout and when we click the logo we get to the homepage so please award points for that if possible. Thanks! @ExpressHermes

Reply:
And i dont want to delay points in this!